src/CMakeLists.txt

```diff
@@ -178,11 +178,27 @@ IF(LMMS_BUILD_WIN32)
 	ENDIF()
 
 	INSTALL(TARGETS lmms RUNTIME DESTINATION "${BIN_DIR}")
-	INSTALL(FILES
+
+	IF(QT5)
+		INSTALL(FILES
+					"${MINGW_PREFIX}/bin/Qt5Core.dll"
+					"${MINGW_PREFIX}/bin/Qt5Gui.dll"
+					"${MINGW_PREFIX}/bin/Qt5Widgets.dll"
+					"${MINGW_PREFIX}/bin/Qt5Xml.dll"
+				DESTINATION .)
+		INSTALL(FILES
+					"${MINGW_PREFIX}/lib/qt5/plugins/platforms/qwindows.dll"
```

Comment:
This fixes `Application failed to start because platform plugin “windows” is missing` message which appears after switching to Qt5 per http://stackoverflow.com/questions/20495620. :+1: